### PR TITLE
chore(starters): Added Start Bootstrap Creative starter

### PR DESCRIPTION
**Pull request by JohnJKerr**
## Description

Added [Start Bootstrap Creative](https://startbootstrap.com/previews/creative/) starter project

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -3747,3 +3747,16 @@
     - Support for Netlify deployment
     - SEO
     - Prettier code styling
+- url: https://johnjkerr.github.io/gatsby-creative/
```
Comment (wardpeet) [inline docs/starters.yml]: @JohnJKerr it seems like your preview is not a gatsby site.

Comment (JohnJKerr) [reply docs/starters.yml]: Hi @wardpeet - How do you mean? It was created from the gatsby-starter-default, and I took the markup from an existing template and converted it to work with gatsby? Then I deployed it to GitHub Pages via GitHub Actions. Is there something I'm missing?

Comment (wardpeet) [reply docs/starters.yml]: I'm looking at https://blackrockdigital.github.io/startbootstrap-creative/ it's not showing that it's a gatsby site. The html is not the same as what a gatsby site will provide. I don't see react being used, .... Are you sure it's the correct upload?

Comment (JohnJKerr) [reply docs/starters.yml]: That's the source markup - my preview is over at https://johnjkerr.github.io/gatsby-creative/

Comment (JohnJKerr) [reply docs/starters.yml]: My bad for linking to the source in the PR description - the URL in the starters.yml is the correct one

Comment (wardpeet) [reply docs/starters.yml]: oh i'm sorry, I clicked on the other link. Thank you!

**Pull request merged**